app/models/manageiq/providers/cloud_manager/orchestration_template_runner.rb

```diff
@@ -96,6 +96,10 @@ def my_signal(no_queue, action, *args, deliver_on: nil, priority: MiqQueue::NORM
     end
   end
 
+  def queue_signal
```

Comment:
@djberg96 I think you still need to handle the deliver_on and priority args since those are passed in by the `my_signal` caller above

Reply:
@agrare Updated.